added a note indicating that the deletion of snippets has to be done using the api (#8327)
magento/devdocs: src/cloud/cdn/cloud-vcl-custom-snippets.md

You can create and manage custom VCL snippets from the Magento Admin UI or by using the Fastly API:

> -  [Magento Admin UI](#manage-custom-vcl-snippets-from-the-magento-admin-ui)—We recommend using the Magento Admin UI to manage custom VCL snippets because it automates the process to validate and upload the custom snippet and apply your changes to the Fastly service configuration. Additionally, you can view and edit the custom VCL snippets added to the Fastly service configuration from the Admin UI. **Please note:** deleting snippets from the Magento UI only removes it from the Magento UI. If you really need a snippet gone you will need to delete it in the Fastly UI. Alternative to deleting a snippet is to set the VCL contents to empty content and upload VCL.

Reviewer: Cloud projects do not get access to Fastly UI. Everything has to be done via the Magento admin. This note needs to be updated to reflect this.

> -  [Magento Admin UI](#manage-custom-vcl-snippets-from-the-magento-admin-ui)—We recommend using the Magento Admin UI to manage custom VCL snippets because it automates the process to validate and upload the custom snippet and apply your changes to the Fastly service configuration. Additionally, you can view and edit the custom VCL snippets added to the Fastly service configuration from the Admin UI. **Please note:** deleting a snippet from the Magento UI only removes it from the Magento UI, to fully delete requires this to be done via the Fastly API. As Cloud projects do not get access to the Fastly UI, it is recommended that an alternative to deleting via the Fastly UI is to set the VCL contents to empty content and upload the VCL via the Magento Admin UI.

Reviewer 2: This is not true. If a customer has an API key, they can delete the snippet themselves using an API call.  

The API call to delete a snippet is documented in the Quick reference section here: https://devdocs.magento.com/cloud/cdn/cloud-vcl-custom-snippets.html#manage-vcl

It is also documented in the Delete Custom snippet section in the individual custom VCL example articles: https://devdocs.magento.com/cloud/cdn/fastly-vcl-badreferer.html#delete-the-custom-vcl-snippet

> -  [Magento Admin UI](#manage-custom-vcl-snippets-from-the-magento-admin-ui)—We recommend using the Magento Admin UI to manage custom VCL snippets because it automates the process to validate and upload the custom snippet and apply your changes to the Fastly service configuration. Additionally, you can view and edit the custom VCL snippets added to the Fastly service configuration from the Admin UI. **Please note:** deleting a snippet from the Magento UI only removes it from the Magento UI, to fully delete requires this to be done via the Fastly API. As Cloud projects do not get access to the Fastly UI, it is recommended that an alternative to deleting via the Fastly UI is to set the VCL contents to empty content and upload the VCL via the Magento Admin UI.

Reviewer 2: ```suggestion
-  [Magento Admin UI](#manage-custom-vcl-snippets-from-the-magento-admin-ui)—We recommend using the Magento Admin UI to manage custom VCL snippets because it automates the process to validate and upload the custom snippet and apply your changes to the Fastly service configuration. Additionally, you can view and edit the custom VCL snippets added to the Fastly service configuration from the Admin UI. 

{:.bs-callout-info}
You must use the Fastly API to [delete custom VCL snippets](https://devdocs.magento.com/cloud/cdn/cloud-vcl-custom-snippets.html#manage-vcl), or you can delete the contents of the snippet in the  Magento Admin UI and upload an empty snippet. See [Delete a custom snippet](https://devdocs.magento.com/cloud/cdn/fastly-vcl-badreferer.html#delete-the-custom-vcl-snippet).
```

> -  [Magento Admin UI](#manage-custom-vcl-snippets-from-the-magento-admin-ui)—We recommend using the Magento Admin UI to manage custom VCL snippets because it automates the process to validate and upload the custom snippet and apply your changes to the Fastly service configuration. Additionally, you can view and edit the custom VCL snippets added to the Fastly service configuration from the Admin UI. **Please note:** deleting snippets from the Magento UI only removes it from the Magento UI. If you really need a snippet gone you will need to delete it in the Fastly UI. Alternative to deleting a snippet is to set the VCL contents to empty content and upload VCL. Additionally, Cloud projects do not get access to the Fastly API. Modification of the same has to be done via the Magento Admin UI.

Reviewer: This feels clearer and less confusing.
```suggestion
-  [Magento Admin UI](#manage-custom-vcl-snippets-from-the-magento-admin-ui)—We recommend using the Magento Admin UI to manage custom VCL snippets because it automates the process to validate and upload the custom snippet and apply your changes to the Fastly service configuration. Additionally, you can view and edit the custom VCL snippets added to the Fastly service configuration from the Admin UI. **Please note:** deleting a snippet from the Magento UI only removes it from the Magento UI, to fully delete requires this to be done via the Fastly API. As Cloud projects do not get access to the Fastly UI, it is recommended that an alternative to deleting via the Fastly UI is to set the VCL contents to empty content and upload the VCL via the Magento Admin UI.
```

Author: Looks good.
